.github/pr-custom-review.yml

```
@@ -14,21 +14,21 @@ rules:
       - ci
       - release-engineering
 
-  - name: Audit rules
-    check_type: changed_files
-    condition:
-      include: ^polkadot/runtime\/(kusama|polkadot|common)\/.*|^polkadot/primitives/src\/.+\.rs$|^substrate/primitives/.*|^substrate/frame/.*
-      exclude: ^polkadot/runtime\/(kusama|polkadot)\/src\/weights\/.+\.rs$|^substrate\/frame\/.+\.md$
-    all_distinct:
-      - min_approvals: 1
-        teams:
-          - locks-review
-      - min_approvals: 1
-        teams:
-          - polkadot-review
-      - min_approvals: 1
-        teams:
-          - srlabs
+  # - name: Audit rules
+  #   check_type: changed_files
+  #   condition:
+  #     include: ^polkadot/runtime\/(kusama|polkadot|common)\/.*|^polkadot/primitives/src\/.+\.rs$|^substrate/primitives/.*|^substrate/frame/.*
+  #     exclude: ^polkadot/runtime\/(kusama|polkadot)\/src\/weights\/.+\.rs$|^substrate\/frame\/.+\.md$
+  #   all_distinct:
+  #     - min_approvals: 1
+  #       teams:
+  #         - locks-review
+  #     - min_approvals: 1
+  #       teams:
+  #         - polkadot-review
+  #     - min_approvals: 1
+  #       teams:
+  #         - srlabs
```

```suggestion
  - name: Audit rules
    check_type: changed_files
    condition:
      include: ^polkadot/runtime\/(kusama|polkadot|common)\/.*|^polkadot/primitives/src\/.+\.rs$|^substrate/primitives/.*
      exclude: ^polkadot/runtime\/(kusama|polkadot)\/src\/weights\/.+\.rs$
    all_distinct:
      - min_approvals: 1
        teams:
          - locks-review
```

As I understood from comments above it should be like this

---

Still it looks restrictive to me as `^substrate/primitives/.*` did not depend on `lock-review` before. I would remove that too, @bkchr @ggwpez wdyt?

---

`locks-review` should only be required when code that is encapsulated in locks is changed:
````
:lock: :lock: 
some code
:lock: :lock: 
```

This requirement should be over the entire repo.

---

I don't think `substrate` should be included in that rule at all.
```suggestion
  - name: Audit rules
    check_type: changed_files
    condition:
      include: ^polkadot/runtime\/(kusama|polkadot|common)\/.*|^polkadot/primitives/src\/.+\.rs$
      exclude: ^polkadot/runtime\/(kusama|polkadot)\/src\/weights\/.+\.rs$
    min_approvals: 1
    teams:
      - locks-review
```